README.md

```diff
@@ -421,3 +421,46 @@ viewport. Any additional `Grid.Col`s will wrap onto the following line.
 **Usage Examples**
 
 [Sample Code](https://github.com/Clever/components/tree/master/docs/GridExample.jsx) ([Live Demo](http://clever.github.io/components/#grid))
+
+### FlexBox and FlexItem
+
+`FlexBox` provides a flex-enabled container as a convenience wrapper around the clever-components flex CSS classes.
+A `FlexBox` may contain any other renderable elements, including other `FlexBox` components or `FlexItem`s.
+
+`FlexItem` similarly provides a convenience wrapper around flex item-specific CSS classes and may in turn contain any combination of React-supported elements.
+
+**NOTE:** Using a `FlexItem` properties have no effect on an element unless it is rendered within a `FlexBox` or other `display: flex` container.
+However `FlexBox` can be used without `FlexItem` in many cases.
+
+#### `<FlexItem />` Options
+
+([Source code](https://github.com/Clever/components/tree/master/src/flex/FlexItem.jsx))
+
+| Prop             | Type     | Description                           | Default
+|------------------|----------|---------------------------------------|---------
+| alignSelf (optional) | ItemAlign | Sets the cross-axis alignment of this flex item only. | ItemAlign.STRETCH
+| children (optional) | React Node | The child items to render in the flex box. | None
+| className (optional) | String | Additional classname to apply to the flex box. | None
+| component (optional) | Any | The tagname or component class for the wrapper component to render for the flex item | "div"
```

Comment:
yeah, will do a pass over this to improve the description a bit.

WRT the prop name, i'm borrowing from the pattern being used in `ReactTransitionGroup`, for example (also used in `react-router`'s `Route` component) - https://facebook.github.io/react/docs/animation.html#rendering-a-different-component

i initially used `wrapperComponent` like i did with the Grid component, but thought it might be useful to be consistent with other patterns out there